src/Phinx/Db/Adapter/MysqlAdapter.php

```diff
@@ -837,6 +839,9 @@ public function getPhinxType($sqlTypeDef)
                     if ($limit === 255) {
                         $limit = null;
                     }
+                    if ($limit === 36) {
+                        $type = static::PHINX_TYPE_UUID;
```

Comment:
There will probably be cases where this may not work, but there isn't really any other way with MySQL but to assume that CHAR(36) is UUID. At least not that I'm aware of.